.github/workflows/ci_cd.yml

```diff
@@ -615,3 +631,31 @@ jobs:
         with:
           cname: ${{ env.DOCUMENTATION_CNAME }}
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  doc-index-stable:
+    name: "Deploy stable docs index"
+    runs-on: ubuntu-latest
+    needs: upload_docs_release
+    steps:
+      - name: "Install Git and clone project"
+        uses: actions/checkout@v4
+      - name: "get version of library"
+        shell: bash
+        run: |
+          echo "FULL_VERSION=$(cat VERSION)" >> $GITHUB_ENV
+
+      - name: "Scrape the stable documentation to PyMeilisearch"
+        run: |
+          VERSION=$(python -c "import os; version=os.environ['FULL_VERSION']; print('.'.join(version.split('.')[:2]))")
+          VERSION_MEILI=$(python -c "import os; version=os.environ['FULL_VERSION']; print('-'.join(version.split('.')[:2]))")
+          echo "Calculated VERSION: $VERSION"
+          echo "Calculated VERSION_MEILI: $VERSION_MEILI"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "VERSION_MEILI=$VERSION_MEILI" >> $GITHUB_ENV
```

Comment:
I think you are missing this in the dev docs index deployment

Reply:
i added it after dev docs deployment.